vm-core/src/trace/trace.rs

```diff
@@ -0,0 +1,40 @@
+use crate::program::REGISTER_NUM;
+use crate::trace::instruction::{Instruction, Opcode};
+use serde::{Serialize, Deserialize};
+use plonky2::field::goldilocks_field::GoldilocksField;
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Step {
+    pub clk: u32,
+    pub pc: u64,
+    // todo delete , use r15
+    pub fp: u64,
```

Comment:
As comment suggested, need to be deleted.

Reply:
ok